### PR TITLE
Your Sport is added

### LINKED ISSUE #290: [New]: Your Sport
### Framework/code-stack

React

### Description

Your Sport is an assessment webpage which asks some questions from users and tell the user's natural sport

### Similar challenges

No, there is no similar challenge

### Screenshots

Screenshot from 2023-10-24 01-14-25

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by Sumitwarrior7**
# Title and Issue number 

Title :Your Sport

Issue No. :#290

Code Stack : React

Close #290

# Checklist:

- [X] I have mentioned the issue number in my Pull Request.
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have updated the Index.html file for my contribution

### CODE REVIEW

File: react/src/machine-coding/your-sport/index.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
+// src/App.js
+import React, { useState } from 'react';
+import './styles.module.css';
```
Comment (sadanandpai) [inline react/src/machine-coding/your-sport/index.jsx:3]: only scoped imports

File: react/src/machine-coding/your-sport/styles.module.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+.my-container {
+  display: flex !important;
+  text-align: center;
+  background-color: #fff;
+  padding: 20px;
+  border-radius: 10px;
+  box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+}
+
+h1 {
```
Comment (sadanandpai) [inline react/src/machine-coding/your-sport/styles.module.css:19]: direct tag modifiers not allowed

File: react/src/machine-coding/your-sport/styles.module.css [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+.answers {
+  display: flex;
+  justify-content: center;
+  gap: 10px;
+  margin-top: 20px;
+}
+
+button {
```
Comment (sadanandpai) [inline react/src/machine-coding/your-sport/styles.module.css:38]: direct tag modifiers not allowed

File: react/src/machine-coding/your-sport/index.jsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+const questions = [
+  'Do you enjoy team sports?',
+  'Do you like running or endurance activities?',
+  'Are you interested in individual sports?',
+  'Do you prefer indoor or outdoor sports?',
+];
+
+const Sport = () => {
```
Comment (sadanandpai) [inline react/src/machine-coding/your-sport/index.jsx:12]: rename to YourSport

**Review by sadanandpai — CHANGES_REQUESTED** [review]

**Comment by Sumitwarrior7** [comment]
The changes have been done , now you can check and merge
@sadanandpai

**Pull request merged**